### PR TITLE
Add codeowners

### PR DESCRIPTION
**Pull request by lkatalin**
@lukehinds I figured you would also want to be added here, but if not just let me know.

### CODE REVIEW

**Review by ashcrow — APPROVED** [review]

**Comment by lkatalin** [comment]
And/or anyone else who would like to be added please let me know! This will ping you on any new pull requests. @ashcrow @ueno

**Comment by ueno** [comment]
@lkatalin I didn't think about it seriously enough, but perhaps it might make sense for me to be added the file to unblock the situations like #299? Or, perhaps the file could simply list the [team name](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax) instead of individuals?

**Comment by lkatalin** [comment]
> @lkatalin I didn't think about it seriously enough, but perhaps it might make sense for me to be added the file to unblock the situations like #299? Or, perhaps the file could simply list the [team name](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax) instead of individuals?

@ueno I'm more than happy to add you. I can do it now. I definitely want to be listed as an individual, personally, to make sure I don't keep missing newly opened PRs.

**Pull request merged**
